torch/distributed/__init__.py

```diff
@@ -23,7 +23,10 @@ def is_available():
         from .rpc import _init_rpc
         from .rpc import *  # noqa: F401
 
-        def init_model_parallel(worker_name, rpc_backend=RpcBackend.PROCESS_GROUP):
+        def init_model_parallel(self_name,
+                                rpc_backend=RpcBackend.PROCESS_GROUP,
+                                self_rank=-1,
+                                rpc_init_url=None):
```

Comment:
Now that you're changing it anyway, can we stick to the same naming as `init_process_group` and use `backend` instead of `rpc_backend` (would you ever need multiple?) and `init_method` instead of `rpc_init_url`.